src/manifest.json

```diff
@@ -9,7 +9,7 @@
             "matches": [
                 "<all_urls>"
             ],
-            "js": ["js/compatibility.js","js/pdf.js","js/pdf.worker.js","js/pdftotext.js","js/bundle.js", "js/blacklist2.js","js/background.js","js/lib/stopwords_res.js", "js/lib/readability.js", "js/relevanttext.js", "js/content.js"]
```

Comment:
Bundle.js isn't a part of the extension. That's the case since the last few times I have worked it. When we say "Load extension" to Chrome, it verifies the files mentioned in manifest.json